CRM/Contribute/BAO/Contribution.php

```diff
@@ -5492,7 +5492,7 @@ public static function createProportionalEntry($entityParams, $eftParams) {
       $paid = $entityParams['line_item_amount'] * ($entityParams['trxn_total_amount'] / $entityParams['contribution_total_amount']);
     }
     // Record Entity Financial Trxn; CRM-20145
-    $eftParams['amount'] = CRM_Contribute_BAO_Contribution_Utils::formatAmount($paid);
+    $eftParams['amount'] = CRM_Utils_Rule::cleanMoney($paid);
```

Comment:
No - we should remove the formatting. By the time it gets this deep it is formatted. cleanMoney actually corrupts the value if called twice on a Euro locale in some cases

Reply:
Wanted to let the test run so the tests passed with this change, now lets see if we rip it out what happens

Reply:
Haven't you ripped out too much now?

Reply:
ah yes fixed but will be interested in that test run anyway